azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -1261,6 +1205,13 @@ public void close() {
     }
     scheduleManager.shutdown();
     executorManager.shutdown();
+    try {
+      server.stop();
+    } catch (Throwable t) {
```

Comment:
Even when an Error is thrown?
see https://www.cis.upenn.edu/~bcpierce/courses/629/jdkdocs/api/java.lang.Error.html
An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch. Most such errors are abnormal conditions. The ThreadDeath error, though a "normal" condition, is also a subclass of Error because most applications should not try to catch it.